### PR TITLE
Update golang.org/x/crypto

### PR DESCRIPTION
**Pull request by johnweldon**

### CODE REVIEW

**Review by cpuschma — APPROVED** [review]

**Comment by johnweldon** [comment]
> @johnweldon Do you want to leave v3/ out because of #467 ?

If we decide to switch to major branch instead of major subdirectory then yes; but we'd probably need some restructuring to get there.  In either case the current state of the repo uses the subdirectory for v3, so I'd keep the changes there, and also in the root for consistency until we do the restructure.

**Comment by cpuschma** [comment]
OK, just wanted to ask since you didn't mirror your changes to the v3 subdirectory 😊

**Comment by johnweldon** [comment]
Derp!

File: v3/go.mod [hunk 1]
```diff
@@ -1,4 +1,4 @@
-module github.com/go-ldap/ldap/v3
+module github.com/go-ldap/ldap
```
Comment (cpuschma) [inline v3/go.mod:1]: 👀 You accidently deleted the v3 module definition

**Review by cpuschma — CHANGES_REQUESTED** [review]

File: v3/go.mod (hunk 1 above)
Comment (johnweldon) [reply v3/go.mod:1]: It's a good thing someone is paying attention

**Review by cpuschma — APPROVED** [review]

**Pull request merged**